DuggaSys/diagram.js

```diff
@@ -3536,11 +3536,19 @@ function addNodes(element)
 {
     var elementDiv = document.getElementById(element.id)
     var nodes = "";
-
-    nodes += "<span class='node mr'></span>";
-    nodes += "<span class='node ml'></span>";
-
+    nodes += "<span id='mr' class='node mr'></span>";
+    nodes += "<span id='ml' class='node ml'></span>";
     elementDiv.innerHTML += nodes;
+
+    var nodeSize = 8*zoomfact;
```

Comment:
Please do not use magic numbers. Make a constant with the value of 8 above `nodeSize` and make a comment why it is 8.